app/Http/Kernel.php

```diff
@@ -42,6 +42,11 @@ class Kernel extends HttpKernel
             'throttle:60,1',
             \Illuminate\Routing\Middleware\SubstituteBindings::class,
         ],
+
+        'loggedin' => [
+            'web',
+            'auth',
+        ],
```

Comment:
The web-Middleware-Group is already assigned to your routes. Yo don't need to assign it again. Therefore it would be better to just assign the auth-Middleware directly to your routes and delete this group.